app/presenters/error_message/detail_collection.rb

```diff
@@ -32,7 +32,7 @@ def short_messages_for(fieldname)
     # when a presenter instance is injected in to govuk_error_summary.
     #
```

Comment:
nit, and only because I went to check what the official documentation said (which I know you wrote anyway) (and yes, I should really have looked for the previous PR)

```suggestion
    # See https://govuk-form-builder.netlify.app/introduction/error-handling/#custom-summary-error-presenter-injection
    #
```